queries/quote.go

```diff
@@ -27,15 +26,13 @@ func scanRows(rows *sql.Rows) ([]*models.Quote, error) {
 
 func InsertQuote(quote models.Quote) error {
 	statement, err := db.DB.Prepare("INSERT INTO quote(quote, character, season, episode, title) VALUES($1, $2, $3, $4, $5)")
-
 	if err != nil {
-		fmt.Print(err)
+		slog.Error(err)
 		return err
 	}
 	defer statement.Close()
 
 	_, err = statement.Exec(quote.Quote, quote.Character, quote.Season, quote.Episode, quote.Title)
-
 	if err != nil {
 		return err
```

Comment:
Include logging here too

```suggestion
		return slog.Error("InsertQuote", "scan rows", err)
```

Reply:
I assume you still want to return the err

> slog.Error("InsertQuote", "scan rows", err) 
> return err

Reply:
Yep, pls return the error